cg/models/demultiplex/flow_cell.py

```diff
@@ -81,6 +81,15 @@ def run_parameters_object(self) -> RunParameters:
             self._run_parameters = RunParameters(run_parameters_path=self.run_parameters_path)
         return self._run_parameters
 
+    @property
+    def sequencer_type(
+        self,
+    ) -> Literal[SequencingInstruments.NOVASEQXPLUS, SequencingInstruments.NOVASEQ6000]:
+        """Return the sequencer type, either 'NovaSeqX' or 'NovaSeq6000'."""
+        if self.machine_name in SequencingInstruments.NOVA_SEQ_6000_NAMES:
+            return SequencingInstruments.NOVASEQ6000
+        return SequencingInstruments.NOVASEQXPLUS
```

Comment:
This will sort of assume that we will always have `NOVASEQXPLUS` if its not `NovaSeq6000` I would add a check to make sure the name is in `NOVA_SEQ_X_PLUS_NAMES`

Reply:
The assumption for now is correct, as only novaseq demultiplexing is done in cg. However, it was decided recently that more sequencing types will be incorporated to cg and this PR does not hold those changes. This will have to be expanded later.